docs/docs/faq.mdx

```diff
@@ -137,7 +137,7 @@ You can override this path using the **SUPERSET_HOME** environment variable.
 Another workaround is to change where superset stores the sqlite database by adding the following in
 `superset_config.py`:
 
-```
+```python
```

Comment:
Things are going to get a lot prettier with all these syntax formatters added! Thanks!